wb/main/models/downloadable_artifacts_model.py

```diff
@@ -14,53 +14,28 @@
       https://software.intel.com/content/dam/develop/external/us/en/documents/intel-openvino-license-agreements.pdf
 """
 import os
-from typing import Tuple, Union
+from typing import Tuple, Optional
 
 from sqlalchemy import Column, Integer, ForeignKey
-from sqlalchemy.orm import relationship, backref, Session
 
 from config.constants import ARTIFACTS_PATH
-from wb.main.enumerates import ArtifactTypesEnum
-from wb.main.utils.utils import get_size_of_files
-from wb.main.models.artifacts_model import ArtifactsModel
-from wb.main.models.enumerates import ARTIFACT_TYPES_ENUM_SCHEMA
-from wb.main.models.jobs_model import JobsModel
+from wb.main.models.shared_artifact_model import SharedArtifactModel
 
 
-class DownloadableArtifactsModel(ArtifactsModel):
-    __tablename__ = 'downloadable_artifacts'
+class DownloadableArtifactsModel(SharedArtifactModel):
 
     __mapper_args__ = {
-        'polymorphic_identity': __tablename__
+        'polymorphic_identity': 'downloadable_artifacts'
```

Comment:
Should it be plural or singular (like `CloudBundleModel`)?

Reply:
It depends on previous type - `__tablename__ == downloadable_artifacts`

Reply:
you are right, fixed